docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: 'FDC3 Openfin Notifications Listener'
+sidebar_label: 'FDC3 Openfin Notifications Listener'
+id: fdc3-openfin-notification-listener
+keywords: [interaction, interactive, interoperability, openfin, fdc3, notifications]
+tags:
+- interaction
+- interactive
+- interoperability
+- openfin
+- notifications
+- fdc3
+- data
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# FDC3 Openfin notifications listener
+
+This is a utility element that pipes notifications to [Openfin notifications](https://developers.openfin.co/of-docs/docs/overview-notifications)
+
+Use cases:
+* Display notifications in the Openfin Notifications panel
+
+## Examples
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+    <TabItem value="genesis">
+        Declaration
+        ```html
+        <rapid-openfin-notification-listener></rapid-openfin-notification-listener>
+        ```
+
+        Usage
+        ```typescript
+        @customElement({
+            name: 'my-element',
+            template: html`
+              <rapid-openfin-notification-listener
+                resource-name="ALL_NOTIFY_ALERT_RECORDS",
+                :notificationsPlatform="${() => ({
+                    id: 'your-platform-id',
+                    icon: 'http://url-to-your-icon.png',
+                    title: 'your platform title'
+                })}"
+                :toastButtons="${(): OpenfinToastButton[] => [
+                    {
+                        task: 'warning-button',
+                        condition: ({details}) => details.NOTIFY_SEVERITY !== 'Warning',
+                        action: ({details, buttonElement, toastElement}) => {
+                            console.log({details, buttonElement, toastElement});
+                            toastElement.close();
+                        },
+                        appearance: 'secondary-orange',
+                        placeholder: 'Dismiss',
+                    },
+                ]}"
+                ></rapid-openfin-notification-listener>
+              `,
+        })
+        export class MyElement extends GenesisElement {}
+        ```
+    </TabItem>
+
+    <TabItem value="react">
+        Declaration
+        ```html
+        <rapid-openfin-notification-listener></rapid-openfin-notification-listener>
+        ```
+
+        Usage
+        ```typescript
+        export function MyComponent() {

```

Comment:
Same comment as other PR regarding to either set the properties directly, or you need to declare the ref